concepticondata/conceptlists/Urban-2011-160.tsv

```diff
@@ -1,161 +1,161 @@
+Urban-2011-160-114	114	eyelash	1540	EYELASH	III	Body Parts and Body Fluids	[]
+Urban-2011-160-115	115	eyelid	17	EYELID	III	Body Parts and Body Fluids	[]
+Urban-2011-160-116	116	finger	1303	FINGER	III	Body Parts and Body Fluids	[]
+Urban-2011-160-117	117	fingernail	1258	FINGERNAIL	III	Body Parts and Body Fluids	[]
+Urban-2011-160-118	118	guts	1334	GUTS	III	Body Parts and Body Fluids	[]
+Urban-2011-160-119	119	heart	1223	HEART	III	Body Parts and Body Fluids	"[{""id"": ""Urban-2011-160-102"", ""name"": ""belly/stomach"", ""polysemy"": 7, ""overt_marking"": 2, ""shift_id"": 22}, {""id"": ""Urban-2011-160-121"", ""name"": ""kidney"", ""polysemy"": 3, ""overt_marking"": 3, ""shift_id"": 26}, {""id"": ""Urban-2011-160-124"", ""name"": ""lungs"", ""polysemy"": 2, ""overt_marking"": 3, ""shift_id"": 28}]"
+Urban-2011-160-120	120	jaw	798	JAW	III	Body Parts and Body Fluids	[]
+Urban-2011-160-121	121	kidney	489	KIDNEY	III	Body Parts and Body Fluids	[]
+Urban-2011-160-122	122	lip	478	LIP	III	Body Parts and Body Fluids	[]
+Urban-2011-160-123	123	liver	1224	LIVER	III	Body Parts and Body Fluids	"[{""id"": ""Urban-2011-160-124"", ""name"": ""lungs"", ""polysemy"": 4, ""overt_marking"": 9, ""shift_id"": 24}]"
+Urban-2011-160-124	124	lungs	688	LUNG	III	Body Parts and Body Fluids	[]
+Urban-2011-160-125	125	mouth	674	MOUTH	III	Body Parts and Body Fluids	"[{""id"": ""Urban-2011-160-122"", ""name"": ""lip"", ""polysemy"": 16, ""overt_marking"": 31, ""shift_id"": 20}, {""id"": ""Urban-2011-160-110"", ""name"": ""cheeck"", ""polysemy"": 2, ""overt_marking"": 3, ""shift_id"": 32}, {""id"": ""Urban-2011-160-5"", ""name"": ""beak"", ""polysemy"": 18, ""overt_marking"": 11, ""shift_id"": 34}, {""id"": ""Urban-2011-160-20"", ""name"": ""estuary"", ""polysemy"": 5, ""overt_marking"": 5, ""shift_id"": 37}]"
+Urban-2011-160-126	126	mucus	208	NASAL MUCUS (SNOT)	III	Body Parts and Body Fluids	[]
+Urban-2011-160-127	127	navel	1838	NAVEL	III	Body Parts and Body Fluids	[]
+Urban-2011-160-128	128	neck	1333	NECK	III	Body Parts and Body Fluids	[]
+Urban-2011-160-129	129	nipple	796	NIPPLE	III	Body Parts and Body Fluids	[]
+Urban-2011-160-130	130	nostrils	1178	NOSTRIL	III	Body Parts and Body Fluids	[]
+Urban-2011-160-131	131	pupil	1658	PUPIL	III	Body Parts and Body Fluids	[]
+Urban-2011-160-132	132	pus	1808	PUS	III	Body Parts and Body Fluids	[]
+Urban-2011-160-133	133	rib	801	RIB	III	Body Parts and Body Fluids	[]
+Urban-2011-160-134	134	saliva/spittle	1503	SALIVA	III	Body Parts and Body Fluids	"[{""id"": ""Urban-2011-160-24"", ""name"": ""foam"", ""polysemy"": 8, ""overt_marking"": 4, ""shift_id"": 35}]"
+Urban-2011-160-135	135	scar	497	SCAR	III	Body Parts and Body Fluids	[]
+Urban-2011-160-136	136	skin	763	SKIN	III	Body Parts and Body Fluids	"[{""id"": ""Urban-2011-160-3"", ""name"": ""bark"", ""polysemy"": 45, ""overt_marking"": 26, ""shift_id"": 33}]"
+Urban-2011-160-137	137	snot	208	NASAL MUCUS (SNOT)	III	Body Parts and Body Fluids	[]
```

Comment:
Both mucus (126) and snot (137) are mapped to NASAL MUCUS (SNOT). Doesn't this affect the concept count?